doc/sphinx/practical-tools/utilities.rst

```diff
@@ -135,7 +135,7 @@ file ``CoqMakefile``. It can contain two kinds of directives.
 
 **Variable assignment**
 
-The variable must belong to the variables listed in the ``Parameters``
+The variable may belong to the variables listed in the ``Parameters``
```

Comment:
@gares Given the comments you made, is this change incorrect?

Reply:
yes